aws-auth-cognito/build.gradle

```diff
@@ -22,4 +22,18 @@ dependencies {
     implementation dependency.androidx.appcompat
     implementation dependency.aws.mobileclient
     implementation dependency.aws.cognitoauth
+
+    testImplementation project(path: ':testutils')
+    testImplementation dependency.junit
+    testImplementation dependency.mockito
```

Comment:
I was surprised that you don't need to include a special second dependency for the `final` value mocking stuff. Isn't there like an `mockito-inline`, or something? Maybe that was an older version....

Reply:
It's a property under resources/mockito-extensions